makefile

```diff
@@ -28,7 +28,7 @@ all: 0-fetch-k8s 1-build-binaries 2-vagrant-up 3-smoke-test 4-e2e-test
 
 1-build-binaries:
 	@chmod +x build.sh
-	@./build.sh $(path)
+	@./build.sh $(PWD)/kubernetes
```

Comment:
This breaks specifying `make path=\kubedir 1` i.e. to the fork where the dev work is being done.